### PR TITLE
Gci 1938 update spring boot

### PR DESCRIPTION
**Pull request by ethatcher83**
Update spring boot deps 

Resolves: GCI-1938

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,6 +65,16 @@
 	</dependencyManagement>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<version>5.4.2</version>
```
Comment (caismith-ch) [inline pom.xml:71]: Looks like you removed the version for this `<junit-jupiter.version>5.4.2</junit-jupiter.version>`

File: pom.xml [hunk 2]
```diff
@@ -166,33 +173,15 @@
...
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-params</artifactId>
-			<version>${junit-jupiter.version}</version>
-			<scope>test</scope>
+			<groupId>org.junit.platform</groupId>
+			<artifactId>junit-platform-commons</artifactId>
+			<version>1.7.0</version>
```
Comment (caismith-ch) [inline pom.xml:178]: Can a version be defined above please

File: pom.xml [hunk 3]
```diff
@@ -166,33 +173,15 @@
...
 		</dependency>
 		<dependency>
 			<groupId>org.junit.platform</groupId>
-			<artifactId>junit-platform-commons</artifactId>
-			<version>1.5.2</version>
+			<artifactId>junit-platform-launcher</artifactId>
+			<version>1.4.0</version>
```
Comment (caismith-ch) [inline pom.xml:183]: Can a version be defined above please

File: pom.xml [hunk 4]
```diff
@@ -46,8 +43,8 @@
 		<jib-maven-plugin.version>3.1.1</jib-maven-plugin.version>
 
 		<!-- Spring -->
-		<spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
-		<spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
+		<spring-boot-dependencies.version>2.4.13</spring-boot-dependencies.version>
```
Comment (caismith-ch) [inline pom.xml:46]: Are there additional errors at version `2.5.5`?

File: pom.xml (hunk 1 above)
Comment (ethatcher83) [reply pom.xml]: thanks, resolved

**Review by caismith-ch — APPROVED** [review]

**Pull request merged**